test/tests/functional/pbs_mom_walltime.py

```diff
@@ -137,3 +137,29 @@ def test_suspend_time_not_counted_in_walltime(self):
         # Used walltime should be less than the sleep time after suspend
         self.server.expect(JOB, {'resources_used.walltime': 10}, op=LE,
                            id=jid1, extend='x')
+
+    def test_mom_restart(self):
+        """
+        Test that time spent on jobs running on MoM will not reset when
+        MoM is restarted
+        """
+
```

Comment:
I think a blank line here isn't necessary but it's not a big deal.

Reply:
fixed